testnet/stacks-node/src/config.rs

```diff
@@ -1085,6 +1106,25 @@ impl NodeConfig {
     }
 }
 
+#[derive(Clone, Debug, Default)]
```

Comment:
The main part of this PR changes the ranking algorithm for the transactions.

Do we know that if you change that algorithm, you still need all these other changes?